### PR TITLE
Export colors and text styles

### PR DESCRIPTION
**Pull request by yigitcanyurtsever**

### CODE REVIEW

File: Zeplin.sketchplugin/Contents/Sketch/manifest.json [hunk 1]
```diff
@@ -13,7 +13,21 @@
...
+        "handler": "exportColors",
+        "icon": "Icons/icZeplinRunner.png",
+        "description": "Export colors to Zeplin."
+    }, {
+        "name": "Export Text Styles…",
+        "identifier": "exportTextStyles",
+        "script": "export.cocoascript",
+        "handler": "exportTextStyles",
+        "icon": "Icons/icZeplinRunner.png",
+        "description": "Export text styles to Zeplin. 🚀"
```
Comment (berkcebi) [inline Zeplin.sketchplugin/Contents/Sketch/manifest.json]: No need for the emoji here. 🤓

File: Zeplin.sketchplugin/Contents/Sketch/export.cocoascript [hunk 2]
```diff
@@ -129,8 +223,8 @@ var onRun = function (context) {
 
     jsonReaderClass = nil;
     readerClass = nil;
-    
-    var assetLibraries = [];
+
+     var assetLibraries = [];
```
Comment (borulday) [inline Zeplin.sketchplugin/Contents/Sketch/export.cocoascript]: is that an indentation issue? 😅

File: Zeplin.sketchplugin/Contents/Sketch/export.cocoascript [hunk 3]
```diff
@@ -201,35 +273,43 @@ var onRun = function (context) {
...
     sketchtoolPath = nil;
-    artboardNames = nil;
     assetLibraries = nil;
-    format = nil;
-    uniqueArtboardSizes = nil;
-    pageIds = nil;
-    artboardIds = nil;
-    
+
+    return directives
```
Comment (borulday) [inline Zeplin.sketchplugin/Contents/Sketch/export.cocoascript]: `;`

**Comment by yigitcanyurtsever** [comment]
@kardeslik Leaving it as a technical debt for now. We can refactor it later on.

**Pull request merged**